### PR TITLE
Release 2024.08.3

### PR DESCRIPTION
**Pull request by thomascavanagh-okta**
## Description:
- **What's changed?** Adding Release Notes for 2024.08.3
- **Is this PR related to a Monolith release?** Yes, 2024.08.3

### Resolves:

* [OKTA-744016](https://oktainc.atlassian.net/browse/OKTA-744016)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md [hunk 1]
```diff
@@ -7,6 +7,20 @@ title: Okta Identity Engine API release notes 2024
...
 
+### Weekly release 2024.08.3
+
+| Change | Expected in Preview Orgs |
+|--------|--------------------------|
+| [Bugs fixed in 2024.08.3](#bugs-fixed-in-2024-08-3)| August 28, 2024 |
+
+#### Bugs fixed in 2024.08.3
+
+* The system log displayed an incorrect actor when events were triggered by a [Reset factor](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/UserLifecycle/#tag/UserLifecycle/operation/resetFactors) API request. (OKTA-752183)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md:18]: thought we were remove links?

**Review by susanharper-okta — COMMENTED** [review]
Thought we were removing links?

File: packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md (hunk 1 above)
Comment (thomascavanagh-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2024-okta-identity-engine/index.md:18]: You said it was optional to remove them so I left the ones which were already included in the blurbs.

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**